### ISSUE TITLE
Feature Request: Include documentation

### ISSUE THREAD

**Issue by tomasz-warniello** [pnprog/goreviewpartner#115]
Could you please include the documentation as a part of this project? I mean this page: http://yuntingdian.com/goreviewpartner/grp-documentation/doc.htm

It would be nice to have it here. Also, I would like to translate it into Polish, and this would resolve the copyright and licensing problem.

Tx.

**Comment by pnprog** [comment]
Hi!
Thanks for the proposal.

I have concerns that once the documentation is part of this project, then we are comited to keep it updated, and I am not so confident with this (even more if there are several translations!). And it would make the source code heavier to download.

But an alternative solution is to have the documentation in a separated github project. Would that be ok with you?

**Comment by tomasz-warniello** [comment]
Sure.

**Comment by pnprog** [comment]
Cool!

By the way; what would an appropriate license for this documentation? I just googled this, and there seems to be a lot of choice available. Maybe the **GNU Free Documentation License (FDL)** ? or one of the **Creative Commons** ? Any recommendation?

**Comment by tomasz-warniello** [comment]
I really don't know.

**Comment by pnprog** [comment]
Ok, so I spent some time reading about those "open source licenses for documentation" and my conclusion is that they are not any better than our good old GPL GNUv3, so I will stick with it.

And so I just created a dedicated [repository](https://github.com/pnprog/grp-documentation) and upload the "source" for the documentation there. The documentation is written using Markdown, and at the moment the rendering by Github seems to have some issues here and there, but that should be easy to fix.

Later I will add some instruction in the README on how to turn this into pdf/html. Maybe a specific "build" folder with the extra few things necessary for that.

You are free to change the organization (file hierarchy) as you think appropriate, to add the Polish translation. Then I will also start working on a French translation as well.

If you see mistake/issue in the English documentation, do not hesitate to fix them, my English is far from perfect !
